Makefile

```diff
@@ -72,3 +72,19 @@ endif
 docs:
 	cd docs && sphinx-build -W -b html -d _build/doctrees . _build/html
 .PHONY: docs
+
+YARN ?= @docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn
+yarn-install:
+	${YARN} install
```

Comment:
Can you please add `.PHONY` to the non-file creating commands

Reply:
What does that mean?

Reply:
TLDR: Phony does not check for file existence when you run make

https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Thank you i will do that